docs/content/tutorials/building-comfort/_index.md

```diff
@@ -0,0 +1,1161 @@
+
+Once the solution is open in VS Code, follow these steps:
+- Press Cmd + Shift + P (on MacOS) or Ctrl + Shift + P (Windows or Linux) to
+    launch the command palette.
+- Select `Dev Containers: Rebuild and Reopen in Container`.
+- Select the `Building Comfort with Drasi` option to launch this tutorial.
+
+{{% /tab %}}
+
+{{% tab header="Local Setup" text=true %}}
+
+You need to have your own Kubernetes cluster setup.
+You can use any Kubernetes setup.
+For a local testing setup, you can choose one of alternatives
+  like Kind, Minikube or k3d.
+
+Make sure that `kubectl` on your system points to your Kubernetes cluster.
+
+{{% /tab %}}
+
+{{% /tabpane %}}
+
+### Setup the Datastore
+For this tutorial, let's imagine that the datastore is a PostgreSQL DB.
+Below is the representation of the schema:
+
+```
++--------------------+      +--------------------+       +--------------------+
+|      Building      |      |       Floor        |       |       Room         |
++--------------------+      +--------------------+       +--------------------+
+| id (PK)            |      | building_id        |       | id (PK)            |
+| name               |      | id (PK)            |       | floor_id           |
++--------------------+      | name               |       | name               |
+                            +--------------------+       | temperature        |
+                                                         | humidity           |
+                                                         | co2                |
+                                                         +--------------------+
+```
+
+Perform the following steps to setup a PostgreSQL database and load it with
+  some sample data.
+
+{{< tabpane >}}
+
+{{% tab header="Dev Container or Codespaces" text=true %}}
+
+The container already has a K3D cluster with postgreSQL service running on it
+  with the port forwarded. Verify this by running `psql` and checking that
+  three tables for Building, Floor and Room exist in the DB.
+
+```sh
+psql
+```
+
+<br />
+
+```
+building-comfort-db=# \dt
+         List of relations
+ Schema |   Name   | Type  | Owner 
+--------+----------+-------+-------
+ public | Building | table | test
+ public | Floor    | table | test
+ public | Room     | table | test
+(3 rows)
+```
+
+{{% /tab %}}
+
+{{% tab header="Local Setup" text=true %}}
+
+You need to have a PostgreSQL service running.
+You can deploy the `postgres.yaml` file located in the `devops/data` directory.
+This YAML file will deploy the service and also create the DB with 3 tables.
+If you want to use an existing postgre instance, you can specify the connection
+parameters in `config.py` located in the same directory.
+
+```sh
+cd devops/data
+kubectl apply -f postgres.yaml
+```
+
+You'll also need to create a port-forward to make the Postgres service
+  accessible on localhost.
+
+```sh
+nohup bash -c 'kubectl port-forward svc/postgres 5432:5432 &'
```

Comment:
Technically we don't really need to use nohup if running locally. The user can just use `kubectl` in a separate terminal.

The issue with this is that you can't use `nohup` on windows. It's probably simpler if we can just stick with kubectl

Reply:
Sure, makes sense.